### PR TITLE
Feat/qa deployment

### PR DESCRIPTION
**Pull request by latch2112**

### CODE REVIEW

File: Portainer/docker-compose.yml [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
+      mode: global
+      placement:
+        constraints: [node.platform.os == linux]
 
   portainer:
     image: portainer/portainer
     command: -H tcp://tasks.agent:9001 --tlsskipverify
     ports:
-      - "9000:9000"
+      - "8080:9000"
```
Comment (nyanofthemoon) [inline Portainer/docker-compose.yml]: ?

File: ES/docker-compose.yml [hunk 2]
```diff
@@ -1,10 +1,12 @@
 version: '3.2'
 services:
   elasticsearch:
+    networks:
+      - proxy
     image: 'elasticsearch:6.7.0'
     command: [ elasticsearch, -E, network.host=0.0.0.0, -E, discovery.zen.ping.unicast.hosts=elasticsearch, -E, discovery.zen.minimum_master_nodes=1 ]
     volumes:
-      - /home/ubuntu/esdata:/usr/share/elasticsearch/data
+      - ./esdata:/usr/share/elasticsearch/data
```
Comment (nyanofthemoon) [inline ES/docker-compose.yml]: ?

**Review by nyanofthemoon — CHANGES_REQUESTED** [review]

File: Portainer/docker-compose.yml (hunk 1 above)
Comment (latch2112) [reply Portainer/docker-compose.yml]: good one

File: ES/docker-compose.yml (hunk 2 above)
Comment (latch2112) [reply ES/docker-compose.yml]: good one

**Review by jecos — APPROVED** [review]

**Review by nyanofthemoon — APPROVED** [review]
🦄

**Pull request merged**